### PR TITLE
More test cases for clipboard API including some edge cases

### PR DESCRIPTION
**Pull request by Sadaf-A**
## Description

This PR adds more test cases for clipoard API including some edge cases

## Changes proposed

 - added test cases for clipboard api

## How to test it

 - Run specs/tests

## Next steps

None.

## Deploy notes

None.

### CODE REVIEW

File: spec/clipboard.spec.js [hunk 1]
```diff
@@ -34,6 +96,39 @@ describe('clipboard.spec: clipboard namespace tests', () => {
...
+                await __close(clipboardText);
+            `);
+            assert.equal(runner.getOutput(), '');
+        });
+        it('returns the same text for multiple consecutive reads', async () => {
+            let exitCode = runner.run(`
+                await Neutralino.clipboard.writeText('Test value');
+                let clipboardText1 = await Neutralino.clipboard.readText();
+                let clipboardText2 = await Neutralino.clipboard.readText();
+                await __close(clipboardText1 === clipboardText2 ? 'ok' : 'not ok');
```
Comment (shalithasuranga) [inline spec/clipboard.spec.js:112]: Do you think it's better to do the comparison from Node with `assert` by sending both strings via an object?

File: spec/clipboard.spec.js [hunk 2]
```diff
@@ -23,7 +23,69 @@ describe('clipboard.spec: clipboard namespace tests', () => {
...
+        it('throws an error if the parameter is an object', async () => {
+            let exitCode = runner.run(`
+                try {
+                    await Neutralino.clipboard.writeText({ key: 'value' });
+                }
+                catch(err) {
+                    await __close(err.code);
+                }
+            `);
+            assert.equal(runner.getOutput(), 'NE_RT_NATRTER');
```
Comment (shalithasuranga) [inline spec/clipboard.spec.js:85]: Seems like we can merge these tests into `throws an error if the parameter is not a string` case?

File: spec/clipboard.spec.js [hunk 3]
```diff
@@ -34,6 +96,39 @@ describe('clipboard.spec: clipboard namespace tests', () => {
...
+            let exitCode = runner.run(`
+                await Neutralino.clipboard.writeText('  Test value  ');
+                let clipboardText = await Neutralino.clipboard.readText();
+                await __close(clipboardText);
+            `);
+            assert.equal(runner.getOutput(), '  Test value  ');
+        });
+        it('returns text with special characters intact', async () => {
+            let exitCode = runner.run(`
+                await Neutralino.clipboard.writeText('@#$%^&*');
```
Comment (shalithasuranga) [inline spec/clipboard.spec.js:126]: Can we add some Unicode chars too?

**Review by shalithasuranga — CHANGES_REQUESTED** [review]
Hello @Sadaf-A. Thanks for sending this. Could you please resolve in-line comments?

Thanks :tada: , let me know if you have questions or ideas

**Comment by Sadaf-A** [comment]
> Hello @Sadaf-A. Thanks for sending this. Could you please resolve in-line comments?
> 
> Thanks :tada: , let me know if you have questions or ideas

Hey! thanks for the review I've addressed the changes requested by you
please have a look

**Review by shalithasuranga — APPROVED** [review]
Thanks so much @Sadaf-A :tada:
